feeds/wlan-ap/opensync/patches/07-conf_changed.patch

```diff
@@ -5,16 +5,58 @@
  #define CMP(cmp, name) \
      (changed |= (changedf->name = ((cmp(conf, state, name, changedf->_uuid)) && \
 -                                   (LOGD("%s: '%s' changed", conf->if_name, #name), 1))))
-+                                   (LOGI("%s: '%s' changed", conf->if_name, #name), 1))))
++                                   (LOGT("%s: '%s' changed", conf->if_name, #name), 1))))
  
  static bool
  wm2_vconf_changed(const struct schema_Wifi_VIF_Config *conf,
-                  const struct schema_Wifi_VIF_State *state,
-                  struct schema_Wifi_VIF_Config_flags *changedf)
- {
-     int changed = 0;
```

Comment:
This should not be removed

Reply:
Fixed the patch.  Not quite sure how it got confused but solved by adding a new patching instead of editing old patch.